typespec-tests/src/main/java/com/cadl/response/ResponseClient.java

```diff
@@ -398,7 +398,7 @@ public boolean exists() {
      */
     @Generated
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public SyncPoller<OperationDetails1, OperationDetails1> beginLroInvalidPollResponse(Resource request) {
+    public SyncPoller<OperationDetails1, Resource> beginLroInvalidPollResponse(Resource request) {
```

Comment:
Diff here is because a slight change in logic. Now if LroMetadata tell us that finalResult need to be from "result" property, we always do that.